discovery-provider/alembic/trigger_sql/handle_follow.sql

```diff
@@ -30,18 +30,49 @@ begin
   -- create a milestone if applicable
   select new_follower_count into milestone where new_follower_count in (10, 25, 50, 100, 250, 500, 1000, 5000, 10000, 20000, 50000, 100000, 1000000);
   if milestone is not null and new.is_delete is false then
-    insert into milestones 
-      (id, name, threshold, blocknumber, slot, timestamp)
-    values
-      (new.followee_user_id, 'FOLLOWER_COUNT', milestone, new.blocknumber, new.slot, new.created_at)
+      insert into milestones 
```

Comment:
I wrote this `insert into milestones` as part of the initial triggers PR.  But it's almost example code atm, I don't think it's actually used anywhere.

I almost wonder if we could drop the `milestones` table in favor of the notification table?

Reply:
yeah I added the migration first to move them all over. Once we know this is working fine, I want to delete the milestones table and replace it with the notifications table in the /notifications route as a follow on task